packages/cells/src/index.ts

```diff
@@ -5,6 +5,7 @@
 
 import '../style/index.css';
 
+export {default as CellDragUtils, ICellTargetArea} from './cellDragUtils';
```

Comment:
Do we intend to continue usage of namespaces? I've heard it isn't recommended, correct me if I'm wrong. E.g http://artsy.github.io/blog/2017/11/27/Babel-7-and-TypeScript/ (see section number 5)

Reply:
We still use namespaces pretty prolifically throughout the JupyterLab codebase. As far as I know, we haven't had any conversations about discouraging their usage.

Reply:
I took that online reference to mean that the Babel ts plugin has a limitation in that it can't handle namespaces, not that they are discouraged in general ts code (we aren't using Babel).

Reply:
I see. One more question - do  they provide any advantages over plain ES6  modules ?